sources/packages/forms/src/form-definitions/partnerinformationandincomeappeal.json

```diff
@@ -584,7 +584,1102 @@
+              "hidden": false,
+              "modalEdit": false,
+              "key": "html91",
+              "tags": [],
+              "properties": {},
+              "conditional": {
+                "show": "",
+                "when": null,
+                "eq": "",
+                "json": ""
+              },
+              "customConditional": "",
+              "logic": [],
+              "attributes": {},
+              "overlay": {
+                "style": "",
+                "page": "",
+                "left": "",
+                "top": "",
+                "width": "",
+                "height": ""
+              },
+              "type": "htmlelement",
+              "input": false,
+              "placeholder": "",
+              "prefix": "",
+              "suffix": "",
+              "multiple": false,
+              "defaultValue": null,
+              "protected": false,
+              "unique": false,
+              "persistent": false,
+              "clearOnHide": true,
+              "refreshOn": "",
+              "redrawOn": "",
+              "tableView": false,
+              "dataGridLabel": false,
+              "labelPosition": "top",
+              "description": "",
+              "errorLabel": "",
+              "tooltip": "",
+              "hideLabel": false,
+              "tabindex": "",
+              "disabled": false,
+              "autofocus": false,
+              "dbIndex": false,
+              "customDefaultValue": "",
+              "calculateValue": "",
+              "calculateServer": false,
+              "widget": null,
+              "validateOn": "change",
+              "validate": {
+                "required": false,
+                "custom": "",
+                "customPrivate": false,
+                "strictDateValidation": false,
+                "multiple": false,
+                "unique": false
+              },
+              "allowCalculateOverride": false,
+              "encrypted": false,
+              "showCharCount": false,
+              "showWordCount": false,
+              "allowMultipleMasks": false,
+              "id": "e4m9daa",
+              "addons": []
+            },
+            {
+              "label": "My spouse/common-law partner's current year income is:",
+              "prefix": "$",
+              "customClass": "font-weight-bold",
+              "mask": false,
+              "spellcheck": true,
+              "tableView": false,
+              "currency": "CAD",
+              "inputFormat": "plain",
+              "validate": {
+                "required": true,
+                "custom": "",
+                "customPrivate": false,
+                "strictDateValidation": false,
+                "multiple": false,
+                "unique": false,
+                "min": "",
+                "max": "",
+                "step": "any",
+                "integer": ""
+              },
+              "key": "currentYearPartnerIncome",
```

Comment:
It is expected to have the same rules as current year student income right? 

![image](https://github.com/user-attachments/assets/a5eba109-5c16-49cc-815c-a16eb8ca804a)

Reply:
For this validation, when you introduce the hidden variable `maxMoneyValue`, please make sure it is added to the top of the form.